compiler/src/main/java/com/squareup/anvil/compiler/ModuleMergerIr.kt

```diff
@@ -291,9 +292,9 @@ internal class ModuleMergerIr(
         )
 
         fun copyArrayValue(name: String) {
-          // TODO This could also be cleaned up with an AnnotationArgumentIr type
-          declaration.clazz.owner.annotation(annotationFqName, scope.fqName)
+          declaration.annotations.find(annotationFqName, scope.fqName).single()
             .argument(name)
+            ?.argument
             ?.second
```

Comment:
Yeah, the Pair is weird here. I really suggest using two arguments in the constructor.